app/lib/mouse.ts

```diff
@@ -0,0 +1,13 @@
+import { setClassEnabled, setOrRemoveStyleProperty } from "./dom";
+
+export const NO_BUTTON_BIT = 0;
+export const LEFT_BUTTON_BIT = 1;
+export const RIGHT_BUTTON_BIT = 2;
+export const MIDDLE_BUTTON_BIT = 4;
+
+export function setCursorOverride(cursor: string | null) {
```

Comment:
I wrote this code a while ago, but I went with the class approach because I wanted to be able to have style rules like

```
body:not(.cursorOverride) .element {
  cursor: whatever;
}
```

Driving the class with React would have been awkward since this is called within a controller class that is performing manual DOM updates.

Reply:
The class approach might make more sense actually:

```
body.cursorOverride[data-cursor="pointer"] * {
  cursor: pointer !important;
}
```

But I am a bit weary of the `*` selector... leaning towards keeping as-is for now since the current code is already using setCursorOverride